images/chromium-headful/Dockerfile

```diff
@@ -146,14 +164,15 @@ RUN set -eux; \
     /home/$USERNAME/.local/share/xorg; \
     chmod 1777 /var/log/neko; \
     chown $USERNAME /var/log/neko/ /tmp/runtime-$USERNAME; \
-    chown -R $USERNAME:$USERNAME /home/$USERNAME; \
-    # clean up
-    apt-get clean -y; \
-    rm -rf /var/lib/apt/lists/* /var/cache/apt/
```

Comment:
should we still run this but maybe further down? or is it not necessary at all?

Reply:
== to @matthewjmarangoni's response after review!